.github/workflows/new-prs.yml

```diff
@@ -34,6 +41,14 @@ jobs:
           chmod a+x github-automation.py
           pip install -r requirements.txt
 
+      # Will be removed shortly, just gathering info to report to Github that
+      # this is not working as expected.
+      - name: Dump Author Association
+        env:
+          AUTHOR_ASSOCIATION: ${{ github.event.pull_request.author_association }}
+        run: |
+          echo "$AUTHOR_ASSOCIATION"
```

Comment:
I did confirm that I am checking the correct `github.event...` attribute by dumping the whole event. Not going to do that here as I don't want to expose any more info than needed (though I expect there wouldn't be any secrets in there anyway).

Reply:
Actually the whole context does contain secrets https://docs.github.com/en/actions/learn-github-actions/contexts#github-context.

So yeah, limiting this to the one thing I need is the right thing to do.

Reply:
https://github.com/llvm/llvm-project/commit/c8655fce45af6f2c13285f218bba83d39c84d5a4